cms/djangoapps/contentstore/views/block.py

```diff
@@ -290,38 +290,23 @@ def can_write(self, course_key):
         return has_studio_write_access(self._user, course_key)
 
 
-class StudioEditModuleRuntime:
+def load_services_for_studio(runtime, user):
     """
-    An extremely minimal ModuleSystem shim used for XBlock edits and studio_view.
-    (i.e. whenever we're not using PreviewModuleSystem.) This is required to make information
+    Function to set some required services used for XBlock edits and studio_view.
+    (i.e. whenever we're not loading preview module system.) This is required to make information
```

Comment:
```suggestion
    (i.e. whenever we're not loading the `_preview_module_system`.) This is required to make information
```

Nit: it's hard to find any references to the "preview module system".

Reply:
Changed